src/test/java/core/basesyntax/AddOperationTest.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;
+
+import static core.basesyntax.service.impl.Operation.BALANCE;
+import static core.basesyntax.service.impl.Operation.RETURN;
+import static core.basesyntax.service.impl.Operation.SUPPLY;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.FruitOperationHandler;
+import core.basesyntax.service.impl.AddOperation;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AddOperationTest {
+    private static FruitOperationHandler fruitOperationHandler;
+    private int expected;
+    private int actual;
+    private FruitRecordDto fruitRecordDto;
+
+    @BeforeClass
+    public static void initialization() {
```

Comment:
Can we initialize our storage here? and in each test where we use storage?

Reply:
We discussed this question in Slack